internal/pkg/service/buffer/dependencies/mocked.go

```diff
@@ -91,16 +91,8 @@ func (v *mocked) StatsCollector() *statistics.CollectorNode {
 
 func (v *mocked) WatcherAPINode() *watcher.APINode {
 	if v.watcherAPINode == nil {
-		// Speedup tests with real clock,
-		// and disable sync interval in tests with mocked clocks,
-		// events will be processed immediately.
-		syncInterval := 10 * time.Millisecond
-		if _, ok := v.Clock().(*clock.Mock); ok {
-			syncInterval = 0
-		}
-
 		var err error
-		v.watcherAPINode, err = watcher.NewAPINode(v, apinode.WithSyncInterval(syncInterval))
+		v.watcherAPINode, err = watcher.NewAPINode(v, apinode.WithSyncInterval(500*time.Millisecond))
```

Comment:
Toto mi komplikovalo upravu.
V testoch sa niektore operacie volali hned, tym, ze sa `interval` nastavil na `0`.

Vyriesil som to v testoch inak.